extensions/transform/org.eclipse.smarthome.transform.scale/src/main/java/org/eclipse/smarthome/transform/scale/internal/ScaleTransformationService.java

```diff
@@ -37,28 +41,50 @@
     /** RegEx to extract a scale definition */
     private static final Pattern LIMITS_PATTERN = Pattern.compile("(\\[|\\])(.*)\\.\\.(.*)(\\[|\\])");
 
+    class OrderedProperties extends Properties {
```

Comment:
I believe we should add the author and goal for this inner class as well.

Reply:
Should this inner class be static I see no reason to have a reference to the the outer class.

Reply:
Should we document that this implementation has limitations, eg iterating over the entryset will not be Ordered, removing elements will cause misery etc.